### ISSUE TITLE
Import updating

### ISSUE THREAD

**Issue by ehdrndd** [BLarzalere/LSTM-Autoencoder-for-Anomaly-Detection#2]
import os
import pandas as pd
import numpy as np
from sklearn.preprocessing import MinMaxScaler
import joblib
#from sklearn.externals import joblib
#import sklearn.external.joblib as extjoblib
import seaborn as sns
sns.set(color_codes=True)
import matplotlib.pyplot as plt
%matplotlib inline
from numpy.random import seed
#from tensorflow import set_random_seed ### tf.set_random_seed(seed) -> tf.random.set_seed(seed)
import tensorflow as tf
#tf.logging.set_verbosity(tf.logging.ERROR)
tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
from keras.layers import Input, Dropout, Dense, LSTM, TimeDistributed, RepeatVector
from keras.models import Model
from keras import regularizers
